federation/pkg/federation-controller/cluster/clustercontroller_test.go

```diff
@@ -37,7 +37,7 @@ import (
 
 func newCluster(clusterName string, serverUrl string) *federation_v1beta1.Cluster {
 	cluster := federation_v1beta1.Cluster{
-		TypeMeta: unversioned.TypeMeta{APIVersion: testapi.Federation.GroupVersion().String()},
+		TypeMeta: unversioned.TypeMeta{APIVersion: registered.GroupOrDie(federation_v1beta1.GroupName).GroupVersion.String()},
```

Comment:
I don't understand why can't this be `federation_v1beta1.SchemeGroupVersion.Version`?